api/util/metrics.go

```diff
@@ -0,0 +1,188 @@
+//
+//   Copyright © 2020 Uncharted Software Inc.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package util
+
+const (
```

Comment:
There are constant definitions in `ta3ta2.go` from `distil-compute` that overlap with these - definitely those for task kewyords and metric names.  We should re-use the existing definitions.

Reply:
Okay replaced the duplicate consts and such with imports from compute where applicable.